torch/fx/traceback.py

```diff
@@ -32,6 +32,19 @@ def set_stack_trace(stack : List[str]):
         current_meta["stack_trace"] = "".join(stack)
 
 
+@compatibility(is_backward_compatible=False)
+def set_seq_nr(seq_nr, bwd=False):
+    global current_meta
+
+    if should_preserve_node_meta:
+        # The seq_nr is captured by eager mode
+        # in the autograd::Node data structure
+        # while the network is being traced.
+        current_meta["seq_nr"] = seq_nr
+        if bwd:
+            current_meta["bwd_op"] = True
+
```

Comment:
I added "bwd_op"  to help create_node() get the sequence_nr from the right place.